package.json

```
@@ -27,92 +27,92 @@
     "url": "https://github.com/UnitedIncome/components.git"
   },
   "devDependencies": {
-    "@babel/core": "*",
-    "@babel/plugin-transform-runtime": "*",
-    "@babel/plugin-transform-spread": "*",
-    "@babel/preset-env": "*",
-    "@babel/preset-react": "*",
-    "@babel/register": "*",
-    "@babel/runtime": "*",
-    "@percy/storybook": "*",
-    "@sambego/storybook-state": "*",
-    "@storybook/addon-a11y": "*",
-    "@storybook/addon-backgrounds": "*",
-    "@storybook/addon-docs": "*",
-    "@storybook/addon-knobs": "*",
-    "@storybook/addon-viewport": "*",
-    "@storybook/react": "*",
-    "accounting": "*",
-    "airbnb-prop-types": "*",
-    "babel-loader": "*",
-    "babel-plugin-istanbul": "*",
-    "babel-plugin-module-resolver": "*",
-    "bootstrap": "*",
-    "classnames": "*",
-    "core-js": "*",
-    "css-loader": "*",
-    "d3-array": "=1.2.4",
-    "d3-interpolate": "*",
-    "d3-scale": "=2.2.2",
-    "d3-shape": "*",
-    "enzyme": "*",
-    "enzyme-adapter-react-16": "*",
-    "eslint": "*",
-    "eslint-config-prettier": "*",
-    "eslint-plugin-jsx-a11y": "*",
-    "eslint-plugin-react": "*",
-    "faucet": "*",
-    "file-loader": "*",
-    "hygen": "*",
-    "ignore-styles": "*",
-    "isomorphic-fetch": "*",
-    "jsdom": "*",
-    "jsdom-global": "*",
-    "lodash": "*",
-    "mini-css-extract-plugin": "*",
-    "moment": "*",
-    "nightwatch": "*",
-    "node-sass": "*",
-    "nyc": "*",
-    "postcss-loader": "*",
-    "postcss-prefixer": "*",
-    "prettier": "*",
-    "prop-types": "*",
-    "query-string": "<6.0.0",
-    "rc-slider": "*",
-    "react": "*",
-    "react-beautiful-dnd": "=10.1.1",
-    "react-dom": "*",
-    "react-required-if": "*",
-    "react-select": "*",
-    "react-spring": "*",
+    "@babel/core": "7.6.4",
+    "@babel/plugin-transform-runtime": "7.6.2",
+    "@babel/plugin-transform-spread": "7.6.2",
+    "@babel/preset-env": "7.6.3",
+    "@babel/preset-react": "7.6.3",
+    "@babel/register": "7.6.2",
+    "@babel/runtime": "7.6.3",
+    "@percy/storybook": "3.2.0",
+    "@sambego/storybook-state": "1.3.6",
+    "@storybook/addon-a11y": "5.2.5",
+    "@storybook/addon-backgrounds": "5.2.5",
+    "@storybook/addon-docs": "5.2.5",
+    "@storybook/addon-knobs": "5.2.3",
+    "@storybook/addon-viewport": "5.2.5",
+    "@storybook/react": "5.2.5",
+    "accounting": "0.4.1",
+    "airbnb-prop-types": "2.15.0",
+    "babel-loader": "8.0.6",
+    "babel-plugin-istanbul": "5.2.0",
+    "babel-plugin-module-resolver": "3.2.0",
+    "bootstrap": "4.3.1",
+    "classnames": "2.2.6",
+    "core-js": "3.3.4",
+    "css-loader": "3.2.0",
+    "d3-array": "1.2.4",
```

Can we keep these with the = so that we know we wanted to version lock it?

```suggestion
    "d3-array": "=1.2.4",
```

---

We can try and see if Dependabot version-locks packages with this format, but I don't know if it will work. Yarn and NPM treat the [`=` operator on a version as optional](https://yarnpkg.com/en/docs/dependency-versions#toc-comparators), so these two formats are effectively the same.

Dependabot has its own internal mechanisms for version-locking using commands against PRs for those packages. Moving away from the "lockfile-only" update approach would require us to use those mechanisms instead since it isn't really going to care what version format we use in `package.json` anymore.

---

Fair enough. I've indicated the locked versions with the suggested syntax.